### ISSUE TITLE
yeoman seems to be `npm` oriented, any support for `pnpm` ?

### ISSUE THREAD

**Issue by maximveksler** [yeoman/yo#779]
Hi,

Documentation, generator-generator are all `npm` focused, especailly with `npm link` structure. 

What is the support scope for `pnpm` based monorepos ?

**Comment by jozefizso** [comment]
Hi @maximveksler, the `yo` and the https://github.com/yeoman/generator-generator are command line tools. While they use `npm` to manage its dependencies, you are free to use another package manager of your choice in your monorepo setup.

**Comment by JoshuaKGoldberg** [comment]
Yup, Yeoman should "Just Work" ✨ with `pnpm`, etc. Related: #651 

Similar to https://github.com/yeoman/yo/issues/651#issuecomment-2610968368: in theory Yeoman works with whatever package manager & system you've got set up. Additionally, older installation commands in `yeoman-generator` have been deprecated in favor of more generalized equivalents: https://github.com/yeoman/generator/issues/1283 & https://github.com/yeoman/generator/issues/1294.

I'm going to go ahead and close this out as aged away. But if someone is having issues getting Yeoman / `yo` to work with pnpm or any of the other new fancy `node_modules/` structures in Yarn/pnpm/etc., please do file a new issue with more details. Cheers! 🎩

**Comment by rimmer** [comment]
Mm, this is on fresh `mcr.microsoft.com/devcontainers/base:jammy` devcontainer image with `ghcr.io/devcontainers-extra/features/pnpm`. 

bash
```
$ pnpm install -g yo generator-office
$ yo office
Error office 

You don't seem to have a generator with the name “generator-office” installed.
But help is on the way:

You can see available generators via npm search yeoman-generator or via http://yeoman.io/generators/. 
Install them with npm install generator-office.

To see all your installed generators run yo --generators. Adding the --help option will also show subgenerators. 

If yo cannot find the generator, run yo doctor to troubleshoot your system.
```

bash
```
$ yo doctor
Yeoman Doctor
Running sanity checks on your system

✔ No .bowerrc file in home directory
✔ Global configuration file is valid
✖ NODE_PATH matches the npm root

Unable to find the npm root, something went wrong.
Try to execute npm -g root --silent on your command-line

✔ No .yo-rc.json file in home directory
✔ Node.js version
✔ npm version
✔ yo version

Found potential issues on your machine :(
```
bash
```
$ npm -g root --silent 
/usr/local/share/nvm/versions/node/v22.14.0/lib/node_modules
echo $NODE_PATH

```

It's quite clear that it is `npm` oriented. That's ok with a docker image, but it doesn't help to diagnose an issue

**Comment by fs-99** [comment]
I can reproduce this too by running `pnpm i -g yo generator-langium` then `yo langium`
